src/iiif/queue_zip_consumer.py

```diff
@@ -133,8 +145,12 @@ def process_message(self, message):
             email_body = render_to_string(
                 "download_zip.html", {"temp_zip_download_url": temp_zip_download_url}
             )
+
             mailing.send_email(record["email_address"], email_subject, email_body)
 
+            remove_blob_from_storage_account(
+                settings.STORAGE_ACCOUNT_CONTAINER_ZIP_QUEUE_JOBS_NAME, job_blob_name
+            )
```

Comment:
Als de code ergens faalt vóór dit punt, dan blijft dat ding voor eeuwig in de storage account zitten. Misschien kan je ook een ttl of iets op de storage account zetten om jobs die ouder zijn dan X aantal dagen automatisch te verwijderen.